src/icons/icons.js

```diff
@@ -1,4 +1,5 @@
-import iconVisibility from "./visibility_24px_rounded.svg";
+import "./icons.css";
+import iconVisibility from "./svgs/visibility_24px_rounded.svg";
```

Comment:
i moved the icon into a directory called `svgs`, since we're going to have dozens of them.